### DISCUSSION TITLE
Anyway to transfer /nsm/elasticsearch data to a new server/location?

### DISCUSSION THREAD

**Discussion by thegreekman** [Security-Onion-Solutions/securityonion#5286] [Unsupported Versions] [answered]
Hello, my /nsm is full and most of the occupied space is coming from /nsm/elasticsearch. I tried to move all the files from /nsm/elasticsearch/nodes/0/indices to a different server but that broke elastic search. Is there any way to backup this data without breaking elasticsearch?

**Comment by dougburks** [comment, marked as answer]
Most folks do something like this:
https://docs.securityonion.net/en/2.3/new-disk.html

**Reply by thegreekman** [reply]
I see, but lets say I want to move this data to another hard drive, sort of like a backup. Is there anyway to bring the data back into elasticsearch without breaking it? I have tried removing data, but it causes shard errors. If I delete the index via curl, that problem is resolved, but I can no longer put the data back. If I do, I get a shard error again.


I basically want to be able to remove data and then import it back at a later date if need be.

**Reply by facyber** [reply]
Hi @thegreekman 

Check out #5074 where I explained in details how can ES backup be done in structural way.

Hope it helps

**Reply by thegreekman** [reply]
Yeah, that helps, thank you.
